x-pack/plugins/aiops/public/components/log_categorization/show_flyout.tsx

```diff
@@ -70,6 +70,7 @@ export async function showCategorizeFlyout(
                     savedSearch={null}
                     selectedField={field}
                     onClose={onFlyoutClose}
+                    embeddingOrigin={'discover_run_pattern_analysis'}
```

Comment:
Unfortunately I don't think hardcoding the ID here is sufficient. This flyout could be (in the future) opened from other places in kibana.
To get the real origin of the action, you'll need to have the value passed into this component as a prop and set by the ui action that has trigged it.
The context that is passed to the action's [execute function](https://github.com/elastic/kibana/blob/354efc2fc8ecc135be3b64c7e4d06e961f2c545b/x-pack/plugins/aiops/public/components/log_categorization/categorize_field_actions.ts#L29) does have an `originatingApp` variable which I added just in case when this was first written.
This has the value `discover` when called from Discover.